app/js/trouble.js

```diff
@@ -1,123 +1,70 @@
-'use strict';
-
-(function (gbl, $) {

```

Comment:
You don't want to modify this part - the code is wrapped in an immediately executing function expression (IIFE) to prevent declaring a bunch of variables in the global namespace.